### ISSUE TITLE
RetryDCV sends invalid data

### ISSUE THREAD

**Issue by zitcomdev** [xolphin/xolphin-api-.net#2]
**TLDR:**
RetryDCV API endpoint expects uppercase values but this SDK does not send uppercase values.

Using .NET Framework 4.7.2 with Newtonsoft.Json v12.0.3 the following error occurs:
The following code

    var client = new Client("[username]", "[password]");
    client.Request.RetryDCV(963062876, new DCV(null, dcv, null))

will result in the following HTTP POST request:

    POST https://api.xolphin.com/v1/requests/963062876/retry-dcv HTTP/1.1
    Authorization: Basic [REDACTED]
    Accept: application/json, text/json, text/x-json, text/javascript, application/xml, text/xml
    User-Agent: Xolphin .NET library v1.6.0
    Content-Type: application/json
    Host: api.xolphin.com
    Content-Length: 60
    Accept-Encoding: gzip, deflate

    {"domain":"[domain]","dcvType":"File","email":null}

The response is as follows:

    {
        "message": "Bad request.",
        "errors": {
            "dcvType": [
                "The DCV type is unknown"
            ]
        }
    }

According to your documentation found on https://api.xolphin.com/docs/v1, the endpoint expects one of the following three values: `EMAIL` `FILE` `DNS` but this SDK sends either `Email` `File` or `DNS`

I see that you already have a fix for this in [Request.cs](https://github.com/xolphin/xolphin-api-.net/blob/c4dde0681584397c15a242daa2142356597e279c/XolphinApiDotNet/Requests/Request.cs#L17). Could you implement something like it in `RequestDCV` and other similar places?

**Comment by XolphinIvo** [comment]
This should be fixed with release v1.6.2
